docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,17 +1,9 @@
 [[modules-remote-clusters]]
 == Remote clusters
 
-ifndef::include-xpack[]
-The _remote clusters_ module enables you to establish uni-directional
-connections to a remote cluster. This functionality is used in
-<<modules-cross-cluster-search,{ccs}>>.
-endif::[]
-ifdef::include-xpack[]
-The _remote clusters_ module enables you to establish uni-directional
-connections to a remote cluster. This functionality is used in
-<<xpack-ccr,{ccr}>> and
-<<modules-cross-cluster-search,{ccs}>>.
-endif::[]
+The _remote clusters_ functionality enables you to establish uni-directional
```

Comment:
Total nit, but unidirectional doesn't need a hyphen.

Reply:
Thanks @debadair. Got that fixed up with 83ffe9ee3d7.